radiation/python/radsw/cldprop.py

```diff
@@ -0,0 +1,610 @@
+    #                             (asy = asycw / ssacw)                     !
+    #    cldfrc - real, cloud fraction of grid mean value              nlay !
+    #    cldfmc - real, cloud fraction for each sub-column       nlay*ngptsw!
+    #                                                                       !
+    #                                                                       !
+    #  explanation of the method for each value of iswcliq, and iswcice.    !
+    #  set up in module "physparam"                                         !
+    #                                                                       !
+    #     iswcliq=0  : input cloud optical property (tau, ssa, asy).        !
+    #                  (used for diagnostic cloud method)                   !
+    #     iswcliq>0  : input cloud liq/ice path and effective radius, also  !
+    #                  require the user of 'iswcice' to specify the method  !
+    #                  used to compute aborption due to water/ice parts.    !
+    #  ...................................................................  !
+    #                                                                       !
+    #     iswcliq=1  : liquid water cloud optical properties are computed   !
+    #                  as in hu and stamnes (1993), j. clim., 6, 728-742.   !
+    #     iswcliq=2  : updated coeffs for hu and stamnes (1993) by aer      !
+    #                  w v3.9-v4.0.                                         !
+    #                                                                       !
+    #     iswcice used only when iswcliq > 0                                !
+    #                  the cloud ice path (g/m2) and ice effective radius   !
+    #                  (microns) are inputs.                                !
+    #     iswcice=1  : ice cloud optical properties are computed as in      !
+    #                  ebert and curry (1992), jgr, 97, 3831-3836.          !
+    #     iswcice=2  : ice cloud optical properties are computed as in      !
+    #                  streamer v3.0 (2001), key, streamer user's guide,    !
+    #                  cooperative institude for meteorological studies,95pp!
+    #     iswcice=3  : ice cloud optical properties are computed as in      !
+    #                  fu (1996), j. clim., 9.                              !
+    #                                                                       !
+    #  other cloud control module variables:                                !
+    #     isubcsw =0: standard cloud scheme, no sub-col cloud approximation !
+    #             >0: mcica sub-col cloud scheme using ipseed as permutation!
+    #                 seed for generating rundom numbers                    !
+    #                                                                       !
+    #  ======================  end of description block  =================  !
+    #
+
+    ds = xr.open_dataset("../lookupdata/radsw_cldprtb_data.nc")
```

Comment:
Can you remind me, how was this netcdf file made? Did you make it? Is the code that did so in this PR? Was this what the Fortran code had hard coded into files? These are numpy arrays correct, so will need some other processing if used in a stencil?

Reply:
I made it by serializing out the data in `radsw_datatb.F` and then writing that to netcdf. Yes, for now these are then read in and converted to storages, as we discussed for lookup tables, until there is a better solution